Documentation/Books/AQL/Examples/MultiplePaths.md

```diff
@@ -2,9 +2,9 @@ Multiple Path Search
 ====================
 
 The shortest path algorithm can only determine one shortest path.
-For example, if this is the full graph:
+For example, if this is the full graph (based on the [mps_graph](../../Manual/Graphs/index.html#the-mps-graph)):
 
-![Example Graph](sp_graph.png)
+![Example Graph](../../Manual/Graphs/mps_graph.png)
```

Comment:
@Simran-B once this is merged and included, can you check back with pdf etc. whether it also picks these images properly? tnx.